internal/store/ingress.go

```diff
@@ -202,6 +202,28 @@ func ingressMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_ingress_status",
+			"Ingress status.",
+			metric.Gauge,
+			basemetrics.STABLE,
```

Comment:
Perhaps should start with an `EXPERIMENTAL` metric https://github.com/kubernetes/kube-state-metrics/blob/main/docs/developer/guide.md#add-new-metrics ?
cc @CatherineF-dev

Reply:
Fixed here https://github.com/kubernetes/kube-state-metrics/pull/2433/commits/c88e3830358a860a0c5dd833c012d4a4ac7ee3fd